readme.html

```diff
@@ -30,19 +39,58 @@ <h2>General Points</h2>
   </tr>
 </table>
 
+<div>
+  <b>NOTE : </b> While running the test connectivity with oAuth, username value is compulsory to pass. The username
```

Comment:
OAuth

Reply:
Have updated  oAuth to OAuth in the document